README.md

```diff
@@ -1,19 +1,23 @@
-# Introducing .NET Multi-platform App UI (MAUI)
+# .NET Multi-platform App UI (.NET MAUI)
 
-.NET Multi-platform App UI (.NET MAUI) is a cross-platform framework for creating native mobile and desktop apps with C# and XAML. Using .NET MAUI, you can develop apps that can run on Android, iOS, macOS, and Windows from a single shared code-base. [Read More](https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui)
+[![Build Status](https://dev.azure.com/xamarin/public/_apis/build/status/MAUI-public?repoName=dotnet%2Fmaui&branchName=main&label=Public)](https://dev.azure.com/xamarin/public/_build/latest?definitionId=57&repoName=dotnet%2Fmaui&branchName=main)[![Build Status](https://devdiv.visualstudio.com/DevDiv/_apis/build/status/MAUI?repoName=dotnet%2Fmaui&branchName=main&label=Private)](https://devdiv.visualstudio.com/DevDiv/_build/latest?definitionId=13330&repoName=dotnet%2Fmaui&branchName=main)
+
+.NET Multi-platform App UI (.NET MAUI) is a cross-platform framework for creating native mobile and desktop apps with C# and XAML. Using .NET MAUI, you can develop apps that can run on Android, iOS, macOS, and Windows from a single shared code-base.
 
 ## Getting Started ##
 
 * [Install .NET MAUI](https://github.com/dotnet/maui/wiki/Getting-Started)
```

Comment:
```suggestion
* [Install .NET MAUI](https://docs.microsoft.com/dotnet/maui/get-started/installation)
```

Reply:
GitHub has some extra steps on creating first project that would have to move to the other page. Both are temporary.